### PR TITLE
cmd/multinode: Removes dependency on deprecated identity-dir flag, code and documentation.

### PR DESCRIPTION
**Pull request by Sembeth**
When the Multinode Dashboard was first released, signed identity files were required. Signed identity files are no longer required. Unsigned identity files are automatically generated with a low level of difficulty. 

Deprecated code that uses the identity-dir was removed.
The documentation was updated.

I have tested the commands "setup", "run", and "add".

## Code Review Checklist (to be filled out by reviewer)
 - [x] NEW: Are there any Satellite database migrations? Are they forwards _and_ backwards compatible? 
 - [x] Does the PR describe what changes are being made?
 - [x] Does the PR describe why the changes are being made?
 - [x] Does the code follow [our style guide](https://github.com/storj/docs/blob/main/code/Style.md)?
 - [x] Does the code follow [our testing guide](https://github.com/storj/docs/blob/main/code/Testing.md)?
 - [x] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [x] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [x] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [x] Does any documentation need updating?
 - [x] Do the database access patterns make sense?

### CODE REVIEW

**Comment by Sembeth** [comment]
I may need a hint why test-sim-backwards.sh fails. The logs are not really helpful.

File: cmd/multinode/main.go [hunk 1]
```diff
@@ -93,22 +92,15 @@ func main() {
 func init() {
 	defaultConfDir := fpath.ApplicationDir("storj", "multinode")
 	cfgstruct.SetupFlag(zap.L(), rootCmd, &confDir, "config-dir", defaultConfDir, "main directory for multinode configuration")
-	cfgstruct.SetupFlag(zap.L(), rootCmd, &identityDir, "identity-dir", "", "main directory for multinode identity credentials")
```
Comment (profclems) [inline cmd/multinode/main.go:96]: I would suggest we still maintain this flag for backward compatibility reasons. We've already marked it deprecated and hidden.

**Review by profclems — CHANGES_REQUESTED** [review]
Thanks for you contribution @Sembeth. This is great! 
However, I think we should maintain the `--identity-dir` flag for backward compatibility reasons, and do nothing with it, i.e. we should still proceed to generating a new identity and warn that the identity is no longer needed.

All the other changes in the dockerfile, documentation, storj-sim and tests are perfect! Maybe we can just revert the changes in main.

**Comment by Sembeth** [comment]
Hi @profclems.
Thank you for the feedback.

I have reverted main.go and peer.go as requested.

You are right that a log message for deprecation should still be included.

Backwards compatibility often makes a lot of sense. I thought I could oversee the current deployments. My assumption was that there is a Docker image and multinode binaries for multiple operating systems.

Docker Image: If someone generated identity files and mounted them in the container and the flag was removed, it doesn't matter because a newer image would start without the identity flag. Therefore, one would simply ignore the mounted identity files and generate a new identity. The worst thing here would be to waste a few files and a few KB of disk space. But here a log message would be helpful.

Binary files: I assumed that most people would just run the binaries without explicitly specifying the identity directory. When they update their binaries and run with their binaries with the identity flag, they should immediately see an error and the allowed flags. I assumed they would figure out fairly quickly that the flag was removed and run the binary without the flag. Here, of course, a log message would give a better hint.

**Review by profclems — APPROVED** [review]

**Review by ifraixedes — APPROVED** [review]

**Pull request merged**